### ISSUE TITLE
[BottomNavigation] Badge value should be read as "%ld item(s)" to match UITabBar

### ISSUE THREAD

**Issue by jverkoey** [material-components/material-components-ios#3803]
https://github.com/material-components/material-components-ios/issues/3181

**Comment by romoore** [comment]
Duplicate
